iam.tf

```diff
@@ -34,10 +34,12 @@ data "aws_iam_policy_document" "task_execution_cloudwatch_access" {
   statement {
     effect = "Allow"
     actions = [
-      "logs:PutRetentionPolicy",
-      "logs:CreateLogGroup"
+      # "logs:PutRetentionPolicy",
+      # "logs:CreateLogGroup",
+      "logs:CreateLogStream",
+      "logs:PutLogEvents"
```

Comment:
For default log group behavior, keep necessary action IAM (`logs:CreateLogGroup`) in place. It would be great to split required actions to separate policies by cw log group use-case (self-manged vs ecs managed).